### PR TITLE
Fix: Dependency Injection Framework breaks Plugin homepage [ED-15411]

### PR DESCRIPTION
**Pull request by Batsirai-muchareva**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/home/classes/transformations-manager.php [hunk 1]
```diff
@@ -35,14 +35,14 @@ class Transformations_Manager {
 	public function __construct( $home_screen_data ) {
 		$container = Plugin::$instance->elementor_container();
 
-		if ( $container->has( Wordpress_Adapter_Interface::class ) ) {
-			$this->wordpress_adapter = $container->get( Wordpress_Adapter_Interface::class );
+		if ( $container->has( Wordpress_Adapter::class ) ) {
```
Comment (baghdasarovelementor) [inline modules/home/classes/transformations-manager.php:38]: Why we need to change this part from Interface to access directly to class? Is there any issue with Interface?

Comment (Batsirai-muchareva) [reply modules/home/classes/transformations-manager.php:38]: In PHP-DI, interfaces are not resolved automatically unless you explicitly define them in the config so I removed them

Comment (baghdasarovelementor) [reply modules/home/classes/transformations-manager.php:38]: Is autowire does not help?

Comment (Batsirai-muchareva) [reply modules/home/classes/transformations-manager.php:38]: No autowire does not resolve interfaces by default. Interfaces can be implemented to multiple concrete classes so DI needs to be told when it's resolving a certain interface which class to resolve

Comment (Batsirai-muchareva) [reply modules/home/classes/transformations-manager.php:38]: try to var_dump `$container->has( Wordpress_Adapter_Interface::class )` to see that it always returns false until you create a config to bind the interface to class

**Review by baghdasarovelementor — APPROVED** [review]

**Review by davseve — APPROVED** [review]

**Pull request merged**